WooCommerce/Classes/ViewRelated/Products/Variations/ProductVariationsViewModel.swift

```diff
@@ -46,6 +46,6 @@ extension ProductVariationsViewModel {
     /// Defines if the More Options button should be shown
     ///
     func shouldShowMoreButton(for product: Product) -> Bool {
-        product.attributesForVariations.isNotEmpty
+        product.variations.isEmpty && product.attributesForVariations.isNotEmpty
```

Comment:
I think this change is causing tests to fail. Could you make sure tests are correctly updated?
Test: `test_more_button_appears_when_product_is_not_empty()`

Reply:
🤦‍♀️ My bad! I'll check the test again.